test/onnx/onnx_test.cpp

```diff
@@ -1657,6 +1658,20 @@ TEST_CASE(logsoftmax_test)
     EXPECT(p == prog);
 }
 
+TEST_CASE(not_test)
```

Comment:
can you move this test to be in alphabetical order? Just to keep consistency for the gen_onnx and onnx_test files

Reply:
otherwise lgtm

Reply:
Moved it, thanks!